### PR TITLE
docs(attribute-directives-dart): Edit Attribute Directives Docs

### PR DESCRIPTION
**Pull request by luisvt**
* edited attribute-directive.jade
* reformat som part of app_component.html
* edited docregion highlight on highlight_directive.dart
* remove extra spaces in highlight_directive_2.dart

### CODE REVIEW

File: public/docs/dart/latest/guide/attribute-directives.jade [hunk 1]
```diff
@@ -1,12 +1,376 @@
...
-  * [Attribute Directives](/docs/ts/latest/guide/attribute-directives.html):
-    The TypeScript version of this chapter
+:marked
+  In this chapter we will
+  * write an attribute directive to change the background color
+  * apply the attribute directive to an element in a template
+  * respond to user-initiated events
+  * pass values into the directive using data binding
+
+  [Live Example](/resources/live-examples/attribute-directives/ts/plnkr.html)
```
Comment (kwalrath) [inline public/docs/dart/latest/guide/attribute-directives.jade]: Please replace this with a link to the Dart source, as in the Template Syntax page.

File: public/docs/dart/latest/guide/attribute-directives.jade [hunk 2]
```diff
@@ -1,12 +1,376 @@
...
+    That wouldn't be nearly as much fun as creating our own directive.
+
+    Besides, we're not just *setting* the color; we'll be *changing* the color
+    in response to a user action, a mouse hover.
+
+.l-main-section
+:marked
+  ## Build a simple attribute directive
+  An attribute directive minimally requires building a controller class annotated with a
+  `Directive` decorator. The `Directive` decorator specifies the selector identifying
```
Comment (kwalrath) [inline public/docs/dart/latest/guide/attribute-directives.jade]: decorator -> attribute (global)

File: public/docs/dart/latest/guide/attribute-directives.jade [hunk 3]
```diff
@@ -1,12 +1,376 @@
...
+:marked
+  We'll extend our directive class with a bindable **input** `highlightColor` property and use it when we highlight text.
+
+  Here is the final version of the class:
+
++makeExample('attribute-directives/dart/lib/highlight_directive.dart', 'class-1', 'lib/highlight_directive.dart (class only)')
+<a id="input"></a>
+:marked
+  The new `highlightColor` property is called an "input" property because data flows from the binding expression into our directive.
+  Notice that we call the `@Input()` decorator function while defining the property.
```
Comment (kwalrath) [inline public/docs/dart/latest/guide/attribute-directives.jade]: call the ... decorator function -> use the ... annotation

**Comment by kwalrath** [comment]
Looks good! Did you view the built version in a browser?

**Comment by luisvt** [comment]
yes, I did

File: public/docs/dart/latest/guide/attribute-directives.jade (hunk 1 above)
Comment (luisvt) [inline public/docs/dart/latest/guide/attribute-directives.jade]: could you give me a link for that Template Syntax page?

Comment (kwalrath) [inline public/docs/dart/latest/guide/attribute-directives.jade]: I just published a wiki page in this repo where you should be able to find that info: [Converting Pages to Dart](https://github.com/angular/angular.io/wiki/Converting-Pages-to-Dart)

**Comment by kwalrath** [comment]
Sorry, I dropped the ball on this. I'll try to take another look today.

**Comment by chalin** [comment]
@luisvt : sorry, but we've adopted a new approach to generating the Dart versions of the dev guide chapters (see #1305) so this PR will not be merged.

On the other hand, I am currently reviewing your code contribution in #1171.

**Comment by kwalrath** [comment]
Closing, per Patrice's comments, but as always thanks very much for your work, @luisvt!

**Pull request closed without merging**